src/app/errors/components/PageNotFound.tsx

```diff
@@ -1,10 +1,52 @@
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
+import styled, { css } from 'styled-components/macro';
+import { SidebarControl } from '../../content/components/Toolbar/styled';
+import theme from '../../theme';
 
-export default () => {
-  return <FormattedMessage id='i18n:error:404'>
-    {(txt) => (
-        <h1>{txt}</h1>
-    )}
-  </FormattedMessage>;
-};
```

Comment:
I'm not really sure if this should be replaced. The `/errors/404` used to render an empty page with just the heading and now it renders the message with toc button even thought the toc doesn't exist there.

Reply:
I've moved this component back to `componenets/Page/` and deleted this one since now we are redirecting everything to `/error/404`